ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -9,9 +9,6 @@ import {
   MetaMetricsEventCategory,
   MetaMetricsEventName,
 } from '../../../../shared/constants/metametrics';
-///: BEGIN:ONLY_INCLUDE_IN(build-main)
-import { showSnapAccountExperimentalToggle } from '../../../../shared/modules/snap-accounts';
```

Comment:
any hints why are we changing all these snaps code here? Seems unrelated to the Terms and Conditions from Blockaid

Reply:
TY. Great catch. Omitted this while resolving git conflict. Fixed https://github.com/MetaMask/metamask-extension/pull/21769/commits/d5e35eff672872a49c97389477c4758132b3d26b